mediawiki/LocalSettings.d/MSC_classification.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
I suggest using another database for security reasons. We should configure that in the private server config repo.